config/level/php/php74.yml

```diff
@@ -1,3 +1,16 @@
 services:
     Rector\Php\Rector\Property\TypedPropertyRector: ~
     Rector\Php\Rector\Assign\NullCoalescingOperatorRector: ~
+
+    Rector\Rector\Function_\FunctionReplaceRector:
+        # https://wiki.php.net/rfc/deprecations_php_7_4#the_real_type
+        is_float: 'is_real'
```

Comment:
@TomasVotruba should this not be the other way arround?

When running rector with `--level php74` my `is_float()`s are replaced by the deprecated `is_real`

Reply:
Yes, you're right! 
Could you send PR to switch them?

Reply:
Yes, you're right! 
Could you send PR to switch them?

Reply:
Sure: https://github.com/rectorphp/rector/pull/1728